src/main/java/co/cask/hydrator/salesforce/plugin/source/streaming/SalesforceReceiver.java

```diff
@@ -72,10 +78,7 @@ private void receive() {
         }
       }
     } catch (InterruptedException e) {
-      LOG.info("Exception in thread", e);
-    } catch (Exception e) {
-      LOG.info("Exception in thread", e);
-      throw e;
+      LOG.error("Exception while receiving messages from pushTopic", e);
```

Comment:
I'm not sure this exception will be printed by something in case if it is not InterruptedException